HW_001/Task-03b.py

```diff
@@ -0,0 +1,26 @@
+# ==============================================================================
+# //////////////////////////////////////////////////////////////////////////////
+# ==============================================================================
+# Задача 6: Вы пользуетесь общественным транспортом? Вероятно, вы
+# расплачивались за проезд и получали билет с номером. Счастливым
+# билетом называют такой билет с шестизначным номером, где сумма
+# первых трех цифр равна сумме последних трех. Т.е. билет с номером
+# 385916 – счастливый, т.к. 3+8+5=9+1+6. Вам требуется написать
+# программу, которая проверяет счастливость билета.
+# ------------------------------------------------------------------------------
+# Решене через обработку строки
+# ------------------------------------------------------------------------------
+number = input('\n Введте номер своей кредитной карты: ')
+depth = len(number)
+sum1 = 0
+sum2 = 0
+r = range(depth // 2)
+for i in r:
+    sum1 = sum1 + int(number[i])
+    sum2 = sum2 + int(number[- (depth // 2) + i])
```

Comment:
Ну вот про это я и говорил. Только разрешите, я немного поправлю.
```suggestion
    sum1 += int(number[i])
    sum2 += int(number[-1 - i])
```